atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,48 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+	
+	private ListaEncadeada lista;
+	private int maxElementos = 10;
+	
+	public PilhaComLista() {
+		this.lista = new ListaEncadeada();
+	}
+	@Override
+	public void push(Integer element) throws Exception {
+		if (isFull()) {
+			throw new Exception("A pilha esta cheia.");
+		}lista.insert(element);
```

Comment:
```suggestion
		}
		lista.insert(element);
```

A indentação do código também é uma boa prática de programação, cada linguagem costuma ter algumas definidas